doc/examples/realtime_servo/realtime_servo_tutorial.rst

```diff
@@ -273,3 +273,28 @@ An example of using the pose commands in the context of servoing to open a door
 .. code-block:: bash
 
     ros2 launch moveit2_tutorials pose_tracking_tutorial.launch.py
+
+Signal Smoothing
+----------------
+
+Reviewing :moveit_codedir:`servo_parameters.yaml <moveit_ros/moveit_servo/config/servo_parameters.yaml>`, notice the parameter `smoothing_filter_plugin_name`. This helps smooth out any irregularities in joint commands to the robot, for example if there is an inconsistent period between commands. It can greatly reduce wear-and-tear on the robot's actuators and, in fact, many robots will not move unless incoming commands are sufficiently smooth.
```

Comment:
```suggestion
Reviewing :moveit_codedir:`servo_parameters.yaml <moveit_ros/moveit_servo/config/servo_parameters.yaml>`, notice the parameter `smoothing_filter_plugin_name`. This helps smooth out any irregularities in joint commands to the robot, for example if there is an inconsistent period between commands. It can greatly reduce wear-and-tear on the robot's actuators and, in fact, many robots will not move unless incoming commands are sufficiently smooth. These are plugins so any user can write their own.
```